cli/pcluster/commands.py

```diff
@@ -198,6 +198,11 @@ def create(args):  # noqa: C901 FIXME!!!
         sys.exit(1)
     except KeyboardInterrupt:
         LOGGER.info("\nExiting...")
+        if bucket_name:
+            LOGGER.info(
+                "Your S3 bucket named %s was not deleted. Do not forget to delete it when deleting your cluster.",
```

Comment:
the bucket is automatically deleted when the cluster is deleted, unless there is an very early failure